src/main/java/org/openapitools/api/impl/SessionsApiServiceImpl.java

```diff
@@ -27,27 +26,21 @@ public Response sessionsPost(Session session, SecurityContext securityContext) t
             LoggedInSession
                 loggedInSession =
                 identity.getSessionService().newSessionWithPassword(session.getUsername(), session.getPassword());
-            String token = loggedInSession.getCredential().toString();
-
-            ApiResponseMessage
-                successMsg =
-                new ApiResponseMessage(Response.Status.CREATED.getStatusCode(), "The session is created successfully");
-            return Response.status(Response.Status.CREATED).entity(successMsg).header("Set-Cookie", token).build();
-
-        } catch (NotAuthorizedException e) {
-            ApiResponseMessage
-                errorMsg =
-                new ApiResponseMessage(Response.Status.UNAUTHORIZED.getStatusCode(),
-                                       "Invalid request: " + e.getMessage());
-            return Response.status(Response.Status.UNAUTHORIZED).entity(errorMsg).build();
-        } catch (Exception e) {
-            ApiResponseMessage
-                errorMsg =
-                new ApiResponseMessage(Response.Status.INTERNAL_SERVER_ERROR.getStatusCode(),
-                                       "Unknown error occurs: " + e.getMessage());
-            return Response.status(Response.Status.INTERNAL_SERVER_ERROR).entity(errorMsg).build();
+            String cookie = loggedInSession.getCredential().toString();
+
+            return Response.status(Response.Status.CREATED).entity("The session is created successfully")
+                .header("Set-Cookie", "V=" + cookie)
```

Comment:
There's a `Response.ResponseBuilder#cookie(NewCookie... cookies)`. We can directly use that.

Reply:
Fixed. Now use `NewCookie cookie = new NewCookie("V", cookieStr);`. Do we need to specify other fields such as domain, path, etc? I think the rest of the fields can be done in the frontend server.

Reply:
Yes, those can be set by the frontend.